lib/experimentUtils.ts

```diff
@@ -42,10 +42,7 @@ export const MESSAGING_EXPERIMENTS_DEFAULT_FEATURES: string[] = [
  *
  * XXX Until we fix the upcase bug, we'll be hiding these dashboards.
```

Comment:
How about changing this comment to just say that if we have experiment dashboards with serious problems, we can hide those dashboards by adding the experiment sluges to this array?